src/website/shared/github.py

```diff
@@ -42,3 +52,96 @@ def get_gh(per_page: int = 30) -> Github:
             )
 
     return Github(auth=gh_auth, per_page=per_page)
+
+
+def create_gh_issue(
+    cached_suggestion: CachedSuggestions,
```

Comment:
Side note, ignore: why is the class name plural? 😮‍💨

Reply:
Yeah, I wondered the same :slightly_smiling_face: